### DISCUSSION TITLE
Recommended AI/ML resources?

### DISCUSSION THREAD

**Discussion by deliquentspacexer** [MarsArtistsCommunity/CommunityOverview#6] [Q&A] [answered]
Hello, just wondering if you could suggest a few resources for learning AI and ML, thank you!

**Comment by oceanbluesky** [comment, marked as answer]
For PI ML specifically:

This remarkably ambitious YT channel: 
https://www.youtube.com/@Eigensteve

https://github.com/tum-pbs/Physics-Based-Deep-Learning 

http://www.databookuw.com/  Chapters 1-4,7, (maybe chapter 9), 11-12

https://maziarraissi.github.io/PINNs/
https://benmoseley.blog/my-research/so-what-is-a-physics-informed-neural-network/

https://neurips.cc/virtual/2023/events/workshop

https://physicsbaseddeeplearning.org/intro.html
